src/providers/ldap/sdap_async.c

```diff
@@ -1803,10 +1802,18 @@ static void generic_ext_search_handler(struct tevent_req *subreq,
 
     ret = sdap_get_generic_ext_recv(subreq, req, &ref_count, &refs);
     talloc_zfree(subreq);
+
     if (ret != EOK) {
-        DEBUG(SSSDBG_OP_FAILURE,
-              "sdap_get_generic_ext_recv failed [%d]: %s\n",
-              ret, sss_strerror(ret));
+        if (ret == ETIMEDOUT) {
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "sdap_async_sys_connect request failed: [%d]: %s "
```

Comment:
The function that is failing is called sdap_get_generic_ext_recv, not sdap_async_sys_connect